features/logging/4.x/elasticsearch.feature

```diff
@@ -0,0 +1,56 @@
+@clusterlogging
+Feature: Elasticsearch related tests
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22050
+  @admin
+  @destructive
+  Scenario: Elasticsearch using dynamic volumes
+    Given I switch to cluster admin pseudo user
+    And default storageclass is stored in the :default_sc clipboard
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    Given I use the "openshift-logging" project
+    And I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/example_unmanaged.yaml |
+    Then the step should succeed
+    And I register clean-up steps:
+    """
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    And I run the :delete client command with:
+      | object_type | pvc  |
+      | all         | true |
```

Comment:
I think you better add `namespace` parameter here so that step properly works in case we end up with unexpected namespace at some point. Thank you.